src/CommandBinaryBuffer.ts

```diff
@@ -166,6 +166,14 @@ class CommandBinaryBuffer extends BinaryBuffer {
     setHours ( hour: number, hours: number ): void {
         this.setUint8(((hours & 0x07) << 5) | (hour & 0x1f));
     }
+
+    getTime (): number {
```

Comment:
please apply these two methods to uplink Time2000 command

Reply:
done